charts/github-actions-runners/README.md

```diff
@@ -0,0 +1,70 @@
+# github-actions-runners
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+A helm chart for managing self hosted github runners utilising [Actions Runner Controller](https://github.com/actions-runner-controller/actions-runner-controller).
+
+
+Further reading on this system:
+[Actions Runner Controller - Github Pages](https://actions-runner-controller.github.io/actions-runner-controller/)
+
+## Prerequisite
+Certificate Manager and Action Replica Controller must be installed as a pre-requisite to this chart.
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.jetstack.io | cert-manager | v1.8.2 |
+| https://actions-runner-controller.github.io/actions-runner-controller| actions-runner-controller | 0.20.2 |
+
+
+## Installing the Chart
+
+Add the repository to Helm:
+
+```bash
+helm repo add snowplow-devops https://snowplow-devops.github.io/helm-charts
+```
+
+Install or upgrading the chart with default configuration:
+
+```bash
+helm upgrade --install github-actions-runners --namespace github-runners --create-namespace --set githubRepo="example/repo" --set runnerNamespace="github-runners" snowplow-devops/github-actions-runners
+```
+
+The default install will use the standard `summerwind/actions-runner` image - should a private image be required this can be enabled via updating the values.yaml
+
+## Uninstalling the Chart
+
+To uninstall/delete the `github-actions-runners` release:
+
+```bash
+helm delete github-actions-runners --namespace github-runners
+```
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://snowplow-devops.github.io/helm-charts | dockerconfigjson | 0.1.0 |
+
+## Configuration
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| dockerconfigjson | object | `{"email":"","name":"snowplow-github-runner-deployment-dockerhub","password":"","server":"https://index.docker.io/v1/","username":""}` | For use with private hosted images on Dockerhub |
```

Comment:
nit: I tend to sort this config section by what appears in values.yaml instead rather than alphabetic.

```
helm-docs -d -s file
```